python-sdk/src/astro/sql/operators/append.py

```diff
@@ -43,7 +44,12 @@ def __init__(
         self.columns = columns or {}
         task_id = task_id or get_unique_task_id("append_table")
 
-        super().__init__(task_id=task_id, **kwargs)
+        super().__init__(
+            task_id=task_id,
+            **kwargs_with_datasets(
+                kwargs=kwargs, input_ds=source_table, output_ds=target_table
```

Comment:
Shouldn't the `input_ds` be both `target_table` and `source_table` in this case?
It feels `target_table` is also an input of append (it is a dataset which is expected to exist before calling this method).

Reply:
I don't think so, `target_table` is just the destination

Reply:
Otherwise, it won't make sense on what happened, 2 inputs and the output is the same as one of the input ds

Reply:
The task does change one of the inputs (table being appended to) - so, from this perspective, the output is one of the inputs (likely modified, considering the `source_table` has content). Otherwise, from a conceptual level, it could seem this task only generates this dataset - without relying on its pre-existence.